### PR TITLE
RSDK-7627, RSDK-7660, RSDK-7480: removes PIL, adds two new methods get_properties and capture_all_from_camera

### PR DESCRIPTION
**Pull request by bhaney**
This PR updates the module to be ready for the viam-sdk version 0.21.0

### CODE REVIEW

File: src/verificationclassifier.py [hunk 1]
```diff
@@ -3,18 +3,16 @@
...
-from PIL import Image
-
-from viam.media.video import RawImage
+from viam.media.video import ViamImage
 
 from viam.module.types import Reconfigurable
 from viam.proto.app.robot import ComponentConfig
-from viam.proto.service.vision import Classification
-from viam.proto.common import ResourceName
+from viam.proto.service.vision import Classification, Detection
```
Comment (kharijarrett) [inline src/verificationclassifier.py]: Do you actually need to import Detection and PointCloudObject?

File: src/verificationclassifier.py [hunk 2]
```diff
@@ -162,6 +160,44 @@ async def do_command(self):
...
+        return_image: bool = False,
+        return_classifications: bool = False,
+        return_detections: bool = False,
+        return_object_point_clouds: bool = False,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> CaptureAllResult:
+        result = CaptureAllResult()
+        if camera_name != self.camera_name:
```
Comment (kharijarrett) [inline src/verificationclassifier.py:186]: I don't think we can do anything about this but I still dislike (even more now) that we need to include camera names in the config.

**Review by kharijarrett — APPROVED** [review]
LGTM! I left small comments

File: src/verificationclassifier.py (hunk 1 above)
Comment (bhaney) [reply src/verificationclassifier.py]: good point - I'll remove them

**Pull request merged**